src/libraries/System.Linq.Expressions/tests/Lambda/LambdaTests.cs

```diff
@@ -943,6 +945,7 @@ public void PrivateDelegate(bool useInterpreter)
 
         [Fact]
         [SkipOnTargetFramework(~TargetFrameworkMonikers.Netcoreapp, "Optimization in .NET Core")]
+        [ActiveIssue("https://github.com/dotnet/runtimelab/issues/155", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
```

Comment:
I think this test (and the one below) are specific to CoreCLR - it would be better to make this ConditionalFact on running on top of coreclr (not Framework, not Mono, not NativeAOT). I think these two would be failing on mono, but they have an ActiveIssue on the entire test class so none of these tests run on Mono.

Reply:
Thanks!